src/space_flight/pipelines/data_engineering/nodes.py

```diff
@@ -101,3 +101,37 @@ def create_master_table(
     master_table = with_companies.drop(["shuttle_id", "company_id"], axis=1)
     master_table = master_table.dropna()
     return master_table
+
+
+# Implementation of a sorting algorithm
+# Source: https://bosker.wordpress.com/2011/04/29/the-worst-algorithm-in-the-world/
+def _slowsort(values, i, j):
```

Comment:
Bad implementation - let me know when you have fixed it!!!!!!
![image](https://user-images.githubusercontent.com/39539935/97364667-7ba06f80-18f8-11eb-87ed-c3e9d336f69f.png)

Reply:
That's !@#$!@#$!@$. This is a great algorithm.  Also   
![image](https://user-images.githubusercontent.com/58553/97381403-1f9c1200-191d-11eb-89c7-093a1bc9d0ba.png)